backend/src/packages/orders/order.repository.ts

```diff
@@ -153,6 +155,75 @@ class OrderRepository implements Omit<IRepository, 'find'> {
       );
   }
 
+  public async findAllUserOrders(
+    search: Partial<{
+      userId: OrderEntityT['userId'];
+      status: OrderEntityT['status'];
+    }>,
+    query: GetPaginatedPageQuery,
+  ): Promise<OrderEntityT[]> {
+    const offset = query.page * query.size;
```

Comment:
We have a helper for this
```suggestion
   const offset = countOffsetByQuery({ size, page });
```